webapp/channels/src/components/widgets/menu/menu_header.tsx

```diff
@@ -14,17 +14,12 @@ type Props = {
 /**
  * @deprecated Use the "webapp/channels/src/components/menu" instead.
  */
-export default class MenuHeader extends React.PureComponent<Props> {
-    public render() {
-        const {children, onClick} = this.props;
+const MenuHeader: React.FC<Props> = ({ children, onClick }) => {
```

Comment:
nit: It is more consistent with the examples to use directly the Props type in the parameters directly, something like:
```suggestion
const MenuHeader = ({ children, onClick }: Props) => {
```